client/my-sites/promote-post-i2/components/tsp-banner/index.tsx

```diff
@@ -44,18 +45,19 @@ function TspBanner( props: TspBannerProps ) {
 									'Reach more people and spark conversations by promoting your content as a native Tumblr post, where users can like, reply, and engage directly with your ad.'
 								) }
 								&nbsp;
-								<ExternalLink
-									href={ localizeUrl(
+								<InlineSupportLink
+									supportContext="blaze_learn_more"
```

Comment:
This context does not exist in the [context links](https://github.com/Automattic/wp-calypso/blob/7ba676615efc425157c6bf729af61005c72a0f44/client/components/inline-support-link/context-links.js) map, so you can delete the `supportContext` prop.

Alternatively, you can include a new entry in the map, and remove the `supportLink` and `supportPostId` props.

Reply:
Man I should have dug deeper. I thought this context was for stats.

Reply:
Actually, it's [also used for stats](https://github.com/Automattic/wp-calypso/blob/04b00775c2447f84dc6fab7682f1a69167103dc1/client/components/inline-support-link/index.jsx#L129), but its main purpose is to avoid having to hardcode links and post IDs everywhere (see https://github.com/Automattic/wp-calypso/pull/54991)

Reply:
Also, it should be `blaze-learn-more` instead of underscores :)

Reply:
haha same thing happened to me. I started with dashed, then saw the JSON and switches to avoid the the quotes when adding the JSON key.